resources-manager/include/IResourceManager.hpp

```diff
@@ -15,13 +15,19 @@
 #include <cstdint>
 
 namespace concord::performance {
+
+struct PruneInfo {
+  uint64_t blocksPerSecond;
```

Comment:
we need to have a lower resolution than per second e.g. 1 block per 5 seconds.
If it will be float we can achieve it, to any other way you think

Reply:
Will change to float